crates/nu-command/tests/commands/first.rs

```diff
@@ -51,7 +51,7 @@ fn gets_first_row_when_no_amount_given() {
 fn gets_first_row_as_list_when_amount_given() {
     let actual = nu!("[1, 2, 3] | first 1 | describe");
 
-    assert_eq!(actual.out, "list<int> (stream)");
+    assert_eq!(actual.out, "list<int>");
```

Comment:
Wondering why this was streaming before and now doesn't

Reply:
Not sure why it was like this before, but now the behavior for inputs and outputs with `first <n>` should be:
- `list -> list`
- `stream -> stream`

For `last <n>` it is:
- `list -> list`
- `stream -> list` (since it already has the last n items in memory)

Reply:
Sounds reasonable to me